exo-add/src/cli.ls

```diff
@@ -30,6 +31,11 @@ module.exports = ->
     throw e
   inquirer.prompt(questions).then (answers) ->
     data := merge data, answers
+    existing-services = []
+    for protection-level of app-config.services
+      if app-config.services[protection-level]
+        existing-services.push Object.keys that
+    check-for-service {existing-services, service-role: data.service-role}
```

Comment:
This logit should be extracted into a method.

```ls
check-for-service service-role: data.service-role, services: app-config.services
```

then in `check-for-services` you massage it using the code from my last comment.